### DISCUSSION TITLE
on.push.branches doesn't work

### DISCUSSION THREAD

**Discussion by kwladyka** [community/community#25471] [Actions]
<p>Here I have workflow.</p>
<p><a href="https://github.com/kwladyka/form-validator-cljs/blob/master/.github/workflows/gh-pages.yaml" rel="nofollow noopener">https://github.com/kwladyka/form-validator-cljs/blob/master/.github/workflows/gh-pages.yaml</a></p>
<pre><code class="lang-auto">
on:  
&amp;nbsp; push:  
&amp;nbsp; &amp;nbsp; branches:  
&amp;nbsp; &amp;nbsp; - doc

</code></pre>
<p>Which should trigger when push to <code>doc</code> branch in this repository. But nothing is happening. I can’t make it to run this workflow.</p>
<p>It looks like it doesn’t monitor <code>doc</code> branch.</p>
<p>When I enter into this workflow in github actions I see “No results matched your search.” and in Branch filter I can choose only master.</p>

**Comment by peter-evans** [comment, marked as answer]
<p>I think the reason it’s not working is because your workflow doesn’t exist in the “doc” branch.</p>

**Comment by kwladyka** [comment]
<p>Yes, I realised that after went to sleep. The structure of YAML <code>on:</code> misslead me. It works more like jenkins, not concourse cicd</p>

**Comment by fabstu** [comment]
<p>This happened to me when I copied on push branch from the docs. It said “main” while my old repo still uses “master”. :slight_smile:</p>
